platform/jobs/edxPlatformAccessibilityMaster.groovy

```diff
@@ -134,8 +134,8 @@ secretMap.each { jobConfigs ->
                    }
                }
            }
-           shell('cd edx-platform')
-           shell('RUN_PA11YCRAWLER=1 ./scripts/accessibility-tests.sh')
+           shell('cd edx-platform \n' +
+                 'RUN_PA11YCRAWLER=1 ./scripts/accessibility-tests.sh')
```

Comment:
Ah, no, sorry. I meant something more like this
`shell('command \n` +
          'other command\n' +
          '....'`
It was this part that struck me as weird 'RUN_PA11YCRAWLER=1 ./scripts/accessibility-tests.sh'